infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/transport/sample/GRPCLogReplicationServerHandler.java

```diff
@@ -103,24 +115,60 @@ public void onCompleted() {
         };
     }
 
+    @Override
+    public StreamObserver<ResponseMsg> reverseReplicate(StreamObserver<RequestMsg> responseObserver) {
+
+        return new StreamObserver<ResponseMsg>() {
+            LogReplicationSession session;
+            @Override
+            public void onNext(ResponseMsg lrResponseMsg) {
+                long requestId = lrResponseMsg.getHeader().getRequestId();
+
+                session = lrResponseMsg.getHeader().getSession();
+
+                try {
+                    sessionToStreamObserverRequestMap.putIfAbsent(session, responseObserver);
+                    log.info("Shama...got the reverseReplicate...hash of oberserver is {}", responseObserver.hashCode());
```

Comment:
your name is in logs

Reply:
Will keep it until we resolve the flaky tests.
My name serves as a good lookup to find the temp logs :D